var/spack/repos/builtin/packages/p4est/package.py

```diff
@@ -10,13 +10,16 @@ class P4est(AutotoolsPackage):
     """Dynamic management of a collection (a forest) of adaptive octrees in
     parallel"""
     homepage = "http://www.p4est.org"
-    url      = "http://p4est.github.io/release/p4est-2.2.tar.gz"
+    url      = "https://github.com/cburstedde/p4est/archive/refs/tags/v2.3.1.tar.gz"
```

Comment:
Love the switch to `https`!

Reply:
Curious. Why not use the URL referenced on the home page as the public repository (i.e., `https://www.github.com/p4est/p4est.github.io/`)?

Reply:
Is it because you aren't making releases there?

Reply:
Yes, I cann't find the latest version in the repository.
And I reviewd its official web, maybe it's better to use the recommended repository.